src/app/types/Setting.ts

```diff
@@ -121,6 +121,9 @@ export interface SapSmartChargingSetting {
   optimizerUrl: string;
   user: string;
   password: string;
+  stickyLimitation: boolean;
+  limitBufferDC: number;
```

Comment:
For the next PR, it would be better to name the unit I think: limitBufferDCPercent